### PR TITLE
Integrate Redesigned Condition List into Lookup Tables

### PR DESCRIPTION
**Pull request by tyler-spangler6**
## What was the problem?

This PR adds functions to read in the new format of the redesigned conditions list and adds it to the existing lookup table for dropdown conditions. 

Associated tickets or Slack threads:
- [#309](https://app.zenhub.com/workspaces/contention-classification-640a24db5dcd08457cebbc79/issues/gh/department-of-veterans-affairs/vagov-claim-classification/309)

## How to test this PR
- Step 1
- Step 2[^secrel]

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

**Review by lukey-luke — COMMENTED** [review]
see comments about using `redesign_...`

File: domain-cc/cc-app/src/python_src/util/lookup_table.py [hunk 1]
```diff
@@ -2,27 +2,47 @@
...
 
 class LookupTable(ABC):
-    """Generalized lookup table for mapping input strings to contention classification codes"""
+    """
+    Generalized lookup table for mapping input strings to contention
+    classification codes
+    """
 
     CSV_FILEPATH = None
+    REDESIGNED_FILEPATH = None
```
Comment (lukey-luke) [inline domain-cc/cc-app/src/python_src/util/lookup_table.py:33]: :pick: why not just describe which csv if there will be an import for each one? e.g. `V2_FILEPATH`??

File: domain-cc/cc-app/src/python_src/util/lookup_table.py [hunk 2]
```diff
@@ -79,3 +144,54 @@ def get_lookup_table(filepath, input_key, output_key):
                 raise
 
     return classification_code_mappings
+
+
+def get_lookup_table(
+    version_num: str,
+    dropdown_v2_filepath: os.path,
+    dropdown_v1_filepath: os.path,
```
Comment (lukey-luke) [inline domain-cc/cc-app/src/python_src/util/lookup_table.py:152]: is `os.path` the intended type?

Screenshot 2023-11-30 at 11 26 05 AM

File: domain-cc/cc-app/src/python_src/util/table_versions.py [hunk 3]
```diff
@@ -0,0 +1,2 @@
+DIAGNOSITC_CODE_TABLE_VERSION = "v0.3Alt"
```
Comment (lukey-luke) [inline domain-cc/cc-app/src/python_src/util/table_versions.py:1]: 👍

**Comment by lukey-luke** [comment]
couple nitpicks, but overall lgtm

**Review by lukey-luke — APPROVED** [review]

**Review by msnwatson — APPROVED** [review]

**Pull request merged**